.github/workflows/build.yml

```diff
@@ -65,49 +76,59 @@ jobs:
               uses: docker/build-push-action@v5
               with:
                   context: .
-                  platforms: linux/amd64,linux/arm64
-                  push: ${{ github.ref == 'refs/heads/develop' }}
+                  platforms: ${{ matrix.platforms }}
+                  push: ${{ github.ref == 'refs/heads/develop' || true }}
```

Comment:
To be reverted ?

Reply:
Yes, best to be reverted, because it I think it won't work when someone does a PR from a fork.

Reply:
Yes, but if I revert now, I won't have the correct image when pulling at compile stage...

Reply:
Reverted.